lib/scarpe/button.rb

```diff
@@ -1,11 +1,46 @@
 # frozen_string_literal: true
 
+# The Scarpe Class
 class Scarpe
+  # The Button class represents a button widget in Scarpe
```

Comment:
I feel like the top two lines -- "The Scarpe Class" and "The Button class represents a button widget in Scarpe" are probably excessive :-)

Reply:
lol yeah but idk what to write there so i just wrote tht.

Reply:
For this, it's probably okay to just not write anything until there's something to write. Somebody seeing "Button < Scarpe::Widget" is gonna figure out that it's a class representing the button as a Scarpe widget.